src/Codeception/Step.php

```diff
@@ -106,6 +107,9 @@ protected function getArgumentsAsString(array $arguments)
 
     protected function parseArgumentAsString($argument)
     {
+        if (class_exists('\Facebook\WebDriver\WebDriverBy') && $argument instanceof \Facebook\WebDriver\WebDriverBy) {
```

Comment:
this can be simplified with `get_class($argument) == 'Facebook\WebDriver\WebDriverBy' `